apps/app/src/styles.css

```diff
@@ -198,3 +198,9 @@ select {
 	--input: 240 3.7% 15.9%;
 	--ring: 142.4 71.8% 29.2%;
 }
+
+@layer base {
+	* {
+		@apply border-border;
```

Comment:
@goetzrobin So i remember having a similar issue with select and I had added this previously but I think you had ended up removing it and done something else. I don't recall how you fixed it do you remember? I also never asked at the time but what was the issue with adding this?

Reply:
Ah ok so I found my original change so I had originally done this because this is what I saw in shadcn repo

* {
	border-color: hsl(var(--border));
}

@goetzrobin removed it and applied border-border to select directly. Your rule is instead just applying that same class in this case so maybe this is will work better?

Linking the change in case it helps jog any memory if there was an issue caused by this https://github.com/thatsamsonkid/spartan/pull/7/files#r1496196592

Reply:
I had updated it to match the recommended global styles on the Shadcn site:

https://ui.shadcn.com/docs/installation/manual

It adds it in a base layer so that it can easily be overridden, but uses it as the default. I can change it to only target the calendar if you prefer.

Reply:
Yea if you want we can scope it till we find out if there's an issue with the above rule. I don't want to put off this PR just because of that but I'm just more curious why we're not able to have this same rule in our project.

Reply:
Yeah originally I did it so border color isn't assigned by default and explicit instead of implicit. Might be more confusing though because it's not in alignment with shadcn?

Reply:
Yeah, my thoughts were to align it with Shadcn, but I can easily switch it if needed, so if anyone has a preference just let me know 😊

Reply:
I agree I think we should align it